### PR TITLE
 #146 Implement transitions alias

### PR DESCRIPTION
**Pull request by Voycawojka**
## Motivation and Context

https://github.com/Knotx/knotx-fragments/issues/146

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: handler/core/handler/common/customTask.conf [hunk 1]
```diff
@@ -0,0 +1,25 @@
+tasks {
```
Comment (tomaszmichalak) [inline handler/core/handler/common/customTask.conf:1]: Do we need those files. I think it is the IDEA problem that it copies those files to the project root folder during tests execution (especially when those test fail).

File: handler/core/src/test/resources/task/factory/taskWithTransitions-alias.conf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# node & transitions
```
Comment (tomaszmichalak) [inline handler/core/src/test/resources/task/factory/taskWithTransitions-alias.conf:1]: This file is valid.

File: handler/core/src/test/java/io/knotx/fragments/task/factory/GraphNodeOptionsTest.java [hunk 3]
```diff
@@ -86,6 +87,16 @@ void expectTransitionSuccessWithNodeBThenNodeC(Vertx vertx) throws Throwable {
     }, vertx);
   }
 
+  @Test
+  @DisplayName("Expect 'on' to alias 'onTransitions'")
```
Comment (tomaszmichalak) [inline handler/core/src/test/java/io/knotx/fragments/task/factory/GraphNodeOptionsTest.java:91]: Please add the case when we mix `onTransitions` and `on`.

Additionally please update all functional tests to use `on` instead od `onTransitions`.

**Comment by Voycawojka** [comment]
Connected with PR: https://github.com/Knotx/knotx-stack/pull/119

**Review by it-getprice-pl — APPROVED** [review]

**Review by tomaszmichalak — APPROVED** [review]

**Pull request merged**